src/dialogs.c

```diff
@@ -37,6 +39,15 @@
 
 #include "pixmaps/logo.xpm"
 
+/*
+ * Schedule a call to the oregano_error() function so that it is
+ * called whenever there are no higher priority events pending.
+ *
+ * The caller should schedule this function call using the GLib
+ * function g_idle_add_full().
+ */
+gboolean oregano_schedule_error (gchar *msg) { oregano_error_with_title (msg, NULL); return FALSE; }
```

Comment:
Wouldn't it better to do the scheduling implicitly? And maybe provide another func `oregano_schedule_error_with_title`?